onedal/cluster/kmeans.py

```diff
@@ -182,7 +183,7 @@ def _init_centroids_custom(
         elif _is_arraylike_not_scalar(init):
             centers = np.asarray(init)
             assert centers.shape[0] == n_clusters
-            assert centers.shape[1] == X_table.column_count
+            assert centers.shape[1] == X_table.get_column_count()
```

Comment:
not pythonic like method, api

Reply:
Yeap, you obviously can do it via python properties. In the other hand - in such a case you will loose naming consistency with C++